Nautilus/Patchers/CraftTreePatcher.cs

```diff
@@ -15,10 +15,18 @@ internal class CraftTreePatcher
     #region Internal Fields
 
     internal static Dictionary<CraftTree.Type, ModCraftTreeRoot> CustomTrees = new();
-    internal static Dictionary<CraftTree.Type, List<Node>> NodesToRemove = new();
-    internal static Dictionary<CraftTree.Type, List<CraftingNode>> CraftingNodes = new();
-    internal static Dictionary<CraftTree.Type, List<TabNode>> TabNodes = new();
-    internal static Dictionary<CraftTree.Type, CraftTree> CachedTrees = new();
```

Comment:
I might be blind but it looks like you remove `CachedTrees` here (not just moved it) and it's used further down on line 93/94.

Reply:
Potentially a consequence of just pulling this one commit from amongst your others

Reply:
Also somehow turned the Dictionaries into lists.. lol

Reply:
looks like it reverted parts of my commit from the other day

Reply:
Yeah that explains both issues, Ramune didn't have those when they made this commit. CachedTrees didn't exist and they were dictionaries at the point when the commit was made

Reply:
I mean they should be dictionaries.